specification/compute/resource-manager/readme.md

```diff
@@ -168,6 +168,17 @@ directive:
       - DefinitionsPropertiesNamesCamelCase
 
 ```
+### Tag: package-2018-10-01

```

Comment:
Why aren't `disks` and `skus` part of this tag?  And should the tag be named different, e.g. see tag `package-2018-06-exclude-gallery`.  Or should there be several different tags for this API version similar to 2018-06-01?

Reply:
It was mistake to exclude disks/skus/gallery in API package specification.. Added them back.. Just that they won't be a part of newer API version..
We will keep tag "package-2018-10-01"..